packages/flutter_driver/lib/src/input.dart

```diff
@@ -35,22 +44,29 @@ class SetInputTextResult extends Result {
   Map<String, dynamic> toJson() => <String, dynamic>{};
 }
 
+/// Submits text entered in a text input widget.
```

Comment:
Done (pointed to https://docs.flutter.io/flutter/material/Input/onSubmitted.html)